src/main/java/ParkingLot/Main.java

```diff
@@ -0,0 +1,77 @@
+package ParkingLot;
+
+import java.util.Scanner;
+
+import static ParkingLot.VehicleParking.*;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        Employees s1 = new SecurityStaff();
```

Comment:
s1 is a bad name for variable

Reply:
fixed, changes pushed to mmykytovych_master